apps/fabric-website/app.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
All the server stuff can be removed. Internally, it's only used for a secure image proxy, and isn't needed (or usable) for the public site.